sdcm/cluster.py

```diff
@@ -3109,20 +3114,22 @@ def start_scylla_monitoring(self, node):
         if node.is_rhel_like() or node.is_ubuntu16():
             run_script = dedent("""
                 cd {0.monitor_install_path}
-                sudo systemctl start docker
                 mkdir -p {0.monitoring_data_dir}
-                chmod 777 {0.monitoring_data_dir}
-                ./start-all.sh -s {0.monitoring_conf_dir}/scylla_servers.yml -n {0.monitoring_conf_dir}/node_exporter_servers.yml -d {0.monitoring_data_dir} -l -v {0.monitoring_version} -b "-web.enable-admin-api"
+                ./start-all.sh \
+                -s {0.monitoring_conf_dir}/scylla_servers.yml \
+                -n {0.monitoring_conf_dir}/node_exporter_servers.yml \
```

Comment:
it looks like it the same command for all OS-s

Reply:
yes, you right. They become the same